test/test_dispatch.py

```diff
@@ -249,13 +255,16 @@ def test_impl_only(self):
         r = self.commute("foo", [
             # m.impl("foo", [](const Tensor& x) { return x })
             lambda m: m.impl_t_t("foo"),
-            # m.impl("foo", torch::kAutograd, [](const Tensor& x) { return x })
-            lambda m: m.impl_t_t("foo", "autograd")
+            # m.impl("foo", torch::kCPU, [](const Tensor& x) { return x })
+            lambda m: m.impl_t_t("foo", "cpu"),
+            # m.impl("foo", torch::kAutogradCPU, [](const Tensor& x) { return x })
+            lambda m: m.impl_t_t("foo", "autogradcpu")
         ])
         self.assertExpectedInline(r, '''\
 name: test::foo
 schema: (none)
-Autograd: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+CPU: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
+AutogradCPU: impl_t_t :: (Tensor _0) -> (Tensor _0) [ boxed unboxed ]
```

Comment:
It would probably be good to actually test the alias here at some point. Especially what happens if you have both "autograd" and "autogradcpu"